spec/person_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './person'
+require './book'
+
+describe Person do
+  before :each do
+    @person = Person.new(30, 'safar')
+  end
+
+  context 'When a new person is created' do
+    it 'should have' do
+      expect(@person.age).to eq 30
+      expect(@person.name).to eq 'safar'
+      expect(@person.rentals).to eq []
+      expect(@person.can_use_services?).to eq true
+    end
+  end
+
+  context 'when a person is created with a different name' do
+    it 'should return the name in constructor' do
+      person = Person.new(45, 'safar')
+      expect(person.correct_name).to eq 'safar'
+    end
+  end
+
+  context 'when a person is created it' do
+    it 'should be able to use services' do
+      person = Person.new(45, 'Kijana Wamalwa')
+      expect(person.can_use_services?).to be true
+    end
+  end
```

Comment:
- [ ] Good job with test coverage, but since we are aiming for a 100% coverage, lets ensure all our methods in all our classes are covered with tests, for example in the `Person Class` we can add tests for the following methods" 
  - `can_use_services?` method,
  - `correct_name ` method,
  - `add_rental`.
#### Please let's ensure all our methods in all our classes are covered with tests.

Reply:
@t-yanick 
the test for can_use_services? is implemented. Isn't it?
waiting for your reply